package.json

```diff
@@ -55,81 +55,82 @@
     "clean-webpack-plugin": "0.1.10",
     "compression": "1.6.2",
     "cookie-parser": "1.4.3",
-    "copy-to-clipboard": "1.1.1",
+    "copy-to-clipboard": "3.0.5",
     "cors": "2.7.1",
-    "crypto-js": "3.1.6",
     "css-loader": "0.23.1",
     "debug": "2.2.0",
-    "dotenv": "1.2.0",
+    "dotenv": "2.0.0",
     "errorhandler": "1.4.3",
     "express": "4.14.0",
     "express-state": "1.4.0",
-    "express-useragent": "0.2.4",
+    "express-useragent": "1.0.4",
     "extract-text-webpack-plugin": "2.0.0-beta.3",
     "file-loader": "0.8.5",
     "fontfaceobserver": "1.7.3",
+    "history": "^3.0.0",
     "html-webpack-plugin": "1.7.0",
     "http-proxy": "1.14.0",
-    "humps": "1.1.0",
+    "humps": "2.0.0",
     "imports-loader": "0.6.5",
-    "jquery": "2.2.4",
     "json-loader": "0.5.4",
     "morgan": "1.7.0",
-    "node-sass": "3.8.0",
+    "node-sass": "4.1.1",
     "normalizr": "2.2.1",
     "pretty-error": "2.0.0",
     "promise": "7.1.1",
     "proxy-middleware": "0.14.0",
     "qs": "6.2.1",
-    "raven": "0.11.0",
+    "raven": "1.1.1",
     "raw-loader": "0.5.1",
-    "react": "0.14.8",
+    "react": "15.4.1",
+    "react-a11y": "0.3.3",
+    "react-addons-create-fragment": "15.4.1",
     "react-bootstrap": "0.29.5",
-    "react-cookie": "0.3.4",
-    "react-dom": "0.14.8",
+    "react-cookie": "1.0.4",
+    "react-dom": "15.4.1",
     "react-helmet": "3.1.0",
+    "react-inlinesvg": "0.5.4",
     "react-metrics": "1.2.1",
-    "react-paginate": "0.4.3",
-    "react-redux": "4.4.5",
-    "react-router": "2.6.1",
+    "react-paginate": "4.0.0",
+    "react-redux": "5.0.1",
+    "react-router": "3.0.0",
     "react-router-bootstrap": "0.20.1",
-    "react-router-redux": "4.0.5",
+    "react-router-redux": "4.0.7",
     "react-router-scroll": "0.2.1",
     "react-scroll": "1.2.0",
     "react-share": "1.11.0",
     "react-sidebar": "2.2.1",
     "redux": "3.5.2",
-    "redux-connect": "2.4.0",
+    "redux-connect": "5.0.0",
     "reselect": "2.5.3",
     "resolve-url": "0.2.1",
-    "sass-loader": "2.0.1",
+    "sass-loader": "4.1.1",
     "serialize-javascript": "1.3.0",
     "serve-favicon": "2.3.0",
     "sitemap": "1.8.1",
     "strip-loader": "0.1.2",
     "style-loader": "0.13.1",
-    "superagent": "1.8.4",
+    "superagent": "3.3.1",
```

Comment:
Definitely for it, although my only fear is it has not had any recent commits. Thoughts?